From u.kleine-koenig@pengutronix.de  Mon Jul 27 09:40:06 2009
From: Uwe Kleine-König <u.kleine-koenig@pengutronix.de>
Date: Thu, 23 Jul 2009 08:33:56 +0200
Subject: Staging: fix typos "man[ae]g?ment" -> "management"
To: Greg KH <gregkh@suse.de>
Cc: Jiri Kosina <jkosina@suse.cz>
Message-ID: <1248330836-14993-1-git-send-email-u.kleine-koenig@pengutronix.de>


This patch was generated by running

	git grep -E -l 'man[ae]g?ment' drivers/staging | xargs -r perl -p -i -e 's/\bman[ae]g?ment\b/management/g'

Signed-off-by: Uwe Kleine-König <u.kleine-koenig@pengutronix.de>
Cc: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/et131x/et1310_mac.c   |    2 +-
 drivers/staging/otus/80211core/cagg.c |    6 +++---
 drivers/staging/otus/80211core/cmm.c  |    4 ++--
 drivers/staging/vt6655/80211mgr.c     |    2 +-
 drivers/staging/vt6655/80211mgr.h     |    2 +-
 drivers/staging/vt6655/power.c        |    2 +-
 drivers/staging/vt6655/power.h        |    2 +-
 drivers/staging/vt6655/wmgr.h         |    4 ++--
 8 files changed, 12 insertions(+), 12 deletions(-)

--- a/drivers/staging/et131x/et1310_mac.c
+++ b/drivers/staging/et131x/et1310_mac.c
@@ -137,7 +137,7 @@ void ConfigMACRegs1(struct et131x_adapte
 	/* Next lets configure the MAC Interface Control register */
 	writel(0, &pMac->if_ctrl.value);
 
-	/* Let's move on to setting up the mii managment configuration */
+	/* Let's move on to setting up the mii management configuration */
 	mii_mgmt_cfg.bits.reset_mii_mgmt = 0;
 	mii_mgmt_cfg.bits.scan_auto_incremt = 0;
 	mii_mgmt_cfg.bits.preamble_suppress = 0;
--- a/drivers/staging/otus/80211core/cagg.c
+++ b/drivers/staging/otus/80211core/cagg.c
@@ -2845,7 +2845,7 @@ u16_t   zfAggSendAddbaRequest(zdev_t* de
 
 
     /*
-     * TBD : Maximum size of managment frame
+     * TBD : Maximum size of management frame
      */
     if ((buf = zfwBufAllocate(dev, 1024)) == NULL)
     {
@@ -3286,7 +3286,7 @@ u16_t   zfAggSendAddbaResponse(zdev_t* d
 
 
     /*
-     * TBD : Maximum size of managment frame
+     * TBD : Maximum size of management frame
      */
     if ((buf = zfwBufAllocate(dev, 1024)) == NULL)
     {
@@ -3439,7 +3439,7 @@ u16_t   zfAggSendBar(zdev_t* dev, TID_TX
 
 
     /*
-     * TBD : Maximum size of managment frame
+     * TBD : Maximum size of management frame
      */
     if ((buf = zfwBufAllocate(dev, 1024)) == NULL)
     {
--- a/drivers/staging/otus/80211core/cmm.c
+++ b/drivers/staging/otus/80211core/cmm.c
@@ -871,7 +871,7 @@ void zfSendMmFrame(zdev_t* dev, u8_t fra
     zmw_declare_for_critical_section();
 
     zm_msg2_mm(ZM_LV_2, "Send mm frame, type=", frameType);
-    /* TBD : Maximum size of managment frame */
+    /* TBD : Maximum size of management frame */
     if ((buf = zfwBufAllocate(dev, 1024)) == NULL)
     {
         zm_msg0_mm(ZM_LV_0, "Alloc mm buf Fail!");
@@ -1099,7 +1099,7 @@ void zfSendMmFrame(zdev_t* dev, u8_t fra
             }
 
             if ((wd->sta.capability[1] & ZM_BIT_0) == 1)
-            {   //spectrum managment flag enable
+            {   //spectrum management flag enable
                 offset = zfStaAddIePowerCap(dev, buf, offset);
                 offset = zfStaAddIeSupportCh(dev, buf, offset);
             }
--- a/drivers/staging/vt6655/80211mgr.c
+++ b/drivers/staging/vt6655/80211mgr.c
@@ -19,7 +19,7 @@
  *
  * File: 80211mgr.c
  *
- * Purpose: Handles the 802.11 managment support functions
+ * Purpose: Handles the 802.11 management support functions
  *
  * Author: Lyndon Chen
  *
--- a/drivers/staging/vt6655/80211mgr.h
+++ b/drivers/staging/vt6655/80211mgr.h
@@ -19,7 +19,7 @@
  * File: 80211mgr.h
  *
  * Purpose: Defines the macros, types, and functions for dealing
- *          with 802.11 managment frames.
+ *          with 802.11 management frames.
  *
  * Author: Lyndon Chen
  *
--- a/drivers/staging/vt6655/power.c
+++ b/drivers/staging/vt6655/power.c
@@ -18,7 +18,7 @@
  *
  * File: power.c
  *
- * Purpose: Handles 802.11 power managment  functions
+ * Purpose: Handles 802.11 power management  functions
  *
  * Author: Lyndon Chen
  *
--- a/drivers/staging/vt6655/power.h
+++ b/drivers/staging/vt6655/power.h
@@ -19,7 +19,7 @@
  *
  * File: power.h
  *
- * Purpose: Handles 802.11 power managment  functions
+ * Purpose: Handles 802.11 power management  functions
  *
  * Author: Lyndon Chen
  *
--- a/drivers/staging/vt6655/wmgr.h
+++ b/drivers/staging/vt6655/wmgr.h
@@ -356,11 +356,11 @@ typedef struct tagSMgmtObject
     BOOL                    bRxBeaconInTBTTWake;
     BYTE                    abyPSTxMap[MAX_NODE_NUM + 1];
 
-    // managment command related
+    // management command related
     UINT                    uCmdBusy;
     UINT                    uCmdHostAPBusy;
 
-    // managment packet pool
+    // management packet pool
     PBYTE                   pbyMgmtPacketPool;
     BYTE                    byMgmtPacketPool[sizeof(STxMgmtPacket) + WLAN_A3FR_MAXLEN];
 
